crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -258,12 +266,17 @@ async fn delete(
             match deleted_row {
                 Ok(0) => Err(not_found!("default config key `{}` doesn't exists", key)),
                 Ok(_) => {
-                    add_config_version(&state, tags, transaction_conn)?;
+                    let version_id = add_config_version(&state, tags, transaction_conn)?;
                     log::info!(
                         "default config key: {key} deleted by {}",
                         user.get_email()
                     );
-                    Ok(HttpResponse::NoContent().finish())
+                    Ok(HttpResponse::NoContent()
```

Comment:
`add_config_version_to_header` can be added here also.
And you can add `add_config_version_to_header` in service_utils so that you don't have to repeat that in the both the crates.

Reply:
both crates have different data types for config_version_header.  one has string , the other has i64

Reply:
And why is it?
Can't both be made of same type?

Reply:
first , that function should not be on service_utils as it is more of handler level function not global level
second , in experiment we are fetching from http headers of (context api response) , so there is not point of first mapping string to i64 , then calling the same function, it does not reduce code in anyway

Reply:
@mahatoankitkumar adding add_config_version_to_header  function is unecessary here , because we aren't doing any checks or typ conversion , its pretty straight forward , just insert it in http_resp